packages/browser/src/transports/fetch.ts

```diff
@@ -9,12 +9,12 @@ import type { BrowserTransportOptions } from './types';
  */
 export function makeFetchTransport(
   options: BrowserTransportOptions,
-  nativeFetch: typeof WINDOW.fetch | undefined = getNativeImplementation('fetch'),
```

Comment:
this type was not what we expected it to be, as `nativeFetch` was interpreted to be `typeof fetch` anyhow because the undefined just lead to it assuming the return type of `getNativeImplementation('fetch')`, which was also `typeof fetch` 😅